packages/palette/src/elements/Carousel/Carousel.tsx

```diff
@@ -206,19 +166,27 @@ export const Carousel: React.FC<CarouselProps> = ({
         </Next>
       </nav>
 
-      <Viewport ref={viewportRef as any}>
-        <Rail style={{ transform: `translateX(${offset})` }}>
-          {cells.map(({ child, ref }, i) => {
-            const isLast = i === cells.length - 1
-
-            return (
-              <Cell key={i} ref={ref} pr={!isLast && CELL_GAP_PADDING_AMOUNT}>
-                {child}
-              </Cell>
-            )
-          })}
-        </Rail>
-      </Viewport>
+      <Wrapper>
```

Comment:
I wouldn't include FullBleed here, which is more of a layout concern. This makes the component less generally useful.

Reply:
Thoughts on it being a shared quality of all v3 comonents?

Reply:
We should just handle it in the wrapper in Force rather than in each situation. In general anything in the design system shouldn't have any outer margins of any kind.